packages/@webex/plugin-cc/src/services/config/types.ts

```diff
@@ -105,6 +140,119 @@ export type DesktopProfileResponse = {
    */
 
   idleCodes: string[];
+
+  /**
+   * Specify dialPlanEnabled of the agent.
```

Comment:
Can we remove "Specify" from all descriptions? I don't think it's required

Reply:
addressed